src/app/image/image-list/image-list.component.html

```diff
@@ -1,6 +1,12 @@
 <div class="image-list">
-    <h3 (click)="navigateToNew()">New</h3>
-    <table-header (onValueChange)='setItemsPerPage($event)' [itemsPerPage]='itemsPerPage'></table-header>
-    <table *ngIf="list && list.length" [dataType]="'image'" [listOfItems]='list' (onSortChange)='setSortOptions($event)' (onNavigate)='navigateToNew()'></table>
+    <div class="table-options">
+        <button (click)="navigateToNew()">New</button>
+        <div class="select-box">
+            <span>Show</span>
+            <table-header (onValueChange)='setItemsPerPage($event)' [itemsPerPage]='itemsPerPage'></table-header>
```

Comment:
Never use single quote in HTML, always use double quotes.
The only time you'd need single quotes is when you want to send a string, but you still wrap it up in double quotes
<div [value]="'Random text'">